### PR TITLE
Update Community Plugin Marketplace docs

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Apply feedback from https://github.com/moussetc/mattermost-plugin-giphy/issues/23#issuecomment-610573128

### CODE REVIEW

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 2. The source code is available in a public git repository.
 3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
 4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
-5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```
Comment (moussetc) [inline site/content/extend/plugins/community-plugin-marketplace.md:20]: If the release_notes_url should point to the specific notes of the current version, it should be precised here.
(Looking at the [mattermost-server code](https://github.com/mattermost/mattermost-server/blob/57717a23afa2917e543d833ceb04634a26c01fd6/model/manifest.go#L114), the doc points to a specific version changelog, however, if a unique changelog page for all versions (such as the Github Releases section, or Mattermost own changelog page) is enough, there's no need to be more explicit in the doc.)

Comment (hanzei) [reply site/content/extend/plugins/community-plugin-marketplace.md:20]: Yes, `release_notes_url` should point to a specific notes.

Does this make it more clear:
```suggestion
5. For the current release and every upcoming ones a changelog has to be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```

Comment (moussetc) [reply site/content/extend/plugins/community-plugin-marketplace.md:20]: Yes, it is very clear, thank you.

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 2. The source code is available in a public git repository.
 3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
 4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
-5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+5. For the current release and every upcoming ones a changelog has to be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```
Comment (aaronrothschild) [inline site/content/extend/plugins/community-plugin-marketplace.md:20]:
```suggestion
5. For the current release and future releases, a changelog must be published. The link to the current release notes should be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```

Comment (hanzei) [reply site/content/extend/plugins/community-plugin-marketplace.md:20]:
```suggestion
5. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
```

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ All community plugins are assigned an _owner_ to guide you through the review pr
 
 ### Process for Updating Community Plugin to the Marketplace
 
-1. Open an issue on the Plugin Marketplace Repository using [a pre-defined template for new plugins](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md). The template contains the checklist above, so you can check the items. Please also point out which commit should be used for the review. You may cut a release candidate (RC) for the reviews.
+1. Open an issue on the Plugin Marketplace Repository using [a pre-defined template for existing plugins](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md). The template contains the checklist above, so you can check the items. Please also point out which commit should be used for the review. You may cut a release candidate (RC) for the reviews.
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:73]:
```suggestion
1. Open an issue on the Plugin Marketplace repository using [a pre-defined template for existing plugins](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md). The template contains the checklist above, so you can check the items. Please also point out which commit should be used for the review. You may cut a release candidate (RC) for the reviews.
```

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
 2. The source code is available in a public git repository.
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:17]:
```suggestion
2. The source code is available in a public GitHub repository.
```

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ Every community plugin must fulfill the following checklist to be added to the M
...
-5. For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+5. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
 6. The plugin has to be out of Beta and be released with at least v1.0.0.
 7. All configuration is accessible via the Mattermost UI.
 8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:23]:
```suggestion
8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 6]
```diff
@@ -70,7 +70,7 @@ All community plugins are assigned an _owner_ to guide you through the review pr
 
 ### Process for Updating Community Plugin to the Marketplace
```
Comment (justinegeffen) [inline site/content/extend/plugins/community-plugin-marketplace.md:71]:
```suggestion
### Process for Updating Community Plugin in the Plugin Marketplace
```

File: site/content/extend/plugins/community-plugin-marketplace.md (hunk 4 above)
Comment (hanzei) [reply site/content/extend/plugins/community-plugin-marketplace.md:17]: `git` is the correct term here. E.g. it should be a GitLab repository

**Pull request merged**